### ISSUE TITLE
Symbolset fonts are broken in Firefox

### ISSUE THREAD

**Issue by erkattak** [madglory/nowplayus#236]
Something is still not right with the fonts.

Chrome on left, Firefox on right
http://d.pr/i/o3Gy

**Comment by mubashariqbal** [comment]
These are all working for me locally, so I'm guessing it either is related to security or how the pipeline is putting all the files together

**Comment by erkattak** [comment]
Yeah it most likely has to do w/ the cross-site loading for assets*.nowplay.us/whatever URLs

**Comment by erkattak** [comment]
https://github.com/rubymaverick/font_assets
